specs/secureboot.table

```diff
@@ -5,7 +5,10 @@ schema([
 ])
 
 extended_schema(DARWIN, [
-    Column("secure_mode", INTEGER, "Secure mode for Intel-based macOS: 0 disabled, 1 full security, 2 medium security"),
+    Column("secure_mode", INTEGER, "(Intel) Secure mode: 0 disabled, 1 full security, 2 medium security"),
+    Column("description", TEXT, "(Apple Silicon) Human-readable description: 'Full Security', 'Reduced Security', or 'Permissive Security'"),
+    Column("allow_kernel_extensions", INTEGER, "(Apple Silicon) Allow user management of kernel extensions from identified developers"),
```

Comment:
Probably doesn't require a change, but I think we use the `allow/deny` convention for flags, we could probably just say `kernel_extensions` as the column name, and then add something like `1 if allowed, 0 otherwise to the description`?

Reply:
I think Sharvil is right -- having `allow_` in the column name feels repetitive. Though we look inconsistent in the spec files